A6httplib/A6httplib.cpp

```diff
@@ -0,0 +1,363 @@
+#include <Arduino.h>
+#include <A6lib.h>
+#include "A6httplib.h"
+
+    // String X_IOTFY_ID = "24580020";
+    // String X_IOTFY_CLIENT = "5dcfb9243fc64d3c1fd9de2a69c5bd";
+    String dummy_string="";
+	String body_rcvd_data = "";
+//	body_rcvd_data.reserve(1000);
+	byte var=A6_NOTOK;
+	String rcv_str="+CIPRCV";
+
+
+
+A6httplib::A6httplib(A6lib *A6l)
+{
+    _A6l = A6l;
+    
+}
+
+
+A6httplib::~A6httplib()
+{
+
+}
+
+
+bool A6httplib::Post(String host, String path, String body)
+{
+     String X_IOTFY_ID = "24580020";
```

Comment:
Hmm, what's this? Should these be here? If not, can you generally clean up the code for release?